k8s/overlays/review/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: ensembl-dev
```

Comment:
Nice to know, thanks! To be consistent, should the namespace key also be removed from all the other kustomization manifests, both in here and other BE repos? And since namespace is often the only directive in the subfolder kustomization files (in addition of including the base folder), can all of these kustomization files be removed (together with their folders) and the gitlab ci/cd config should then just use the base customization in all the deployment jobs in all the environments?